src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -1342,6 +1386,64 @@ public static async Task<Stream> ReadAsStreamUncompressedAsync(HttpResponseMessa
             return true;
         }
 
+        private async Task Broadcast<TResult>(RavenCommand<TResult> command, SessionInfo sessionInfo, CancellationToken token)
+        {
+            if (command is IRaftCommand == false)
+                throw new InvalidOperationException("You can broadcast only raft commands.");
+
+            var tasks = new Dictionary<Task, CancellationTokenSource>();
+
+            command.SetTimeout(_defaultBroadcastTimeout ?? TimeSpan.FromSeconds(30));
```

Comment:
Maybe it would be better to set default broadcast timeout in conventions to this value instead of introducing nullable?

Reply:
done